core-modules/discovery-workshop/mapping.md

```diff
@@ -0,0 +1,25 @@
+---
+description: >-
+  An exercise intended to help Product Managers find a succinct way for their users to reach their goal.
+---
+
+# Mapping
+
+### Introduction
+
+Mapping serves as a way to make clear the path our users take to find a service, understand it and acheive their goals. This exercise gives a clear indication of where the common steps are for each organisation participating in the Design Sprint. The path should be guided by three categories: Discovery, Learning and Using.
+
+Discovery is how the user finds the service for the first time.
+Learning is the process of understanding the services offered by the charity.
+Using is the steps towards acheiving the goal or at least getting closer to the goal.
+
+### Required Materials
+
+A common goal, [user personas](./user-personas/README.md), post-its, pens
+
```

Comment:
It may be worth adding something like 'Digital whiteboarding service such as Miro if working remotely online' to this section

Reply:
Yeah, this is a great point - it's probably worth adding a section to all the other ones also on how to run remotely. The format all comes from having planned everything in person back on TvsA